### PR TITLE
feat(portal):  support two login options

### PR DESCRIPTION
**Pull request by yangpangpangya**
## Description

if user login as a reseller , should use reseller auth0
if user login as a customer , should use customer auth0
add other env variables for reseller company setting
update login page 
update app name and favicon
## Screenshots (if appropriate)
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] CI/CD or documentation update (changes to CI/CD pipeline or documentation)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Review by btngoc-cmcglobal — APPROVED** [review]
approved

**Review by btngoc-cmcglobal — APPROVED** [review]

File: vortex-app/vortex-app-portal/.env.production [hunk 1]
```diff
@@ -3,4 +3,15 @@ AUTH0_DOMAIN=RUNTIME_AUTH0_DOMAIN
 AUTH0_CLIENT_ID=RUNTIME_AUTH0_CLIENT_ID
 AUTH0_AUDIENCE=RUNTIME_AUTH0_AUDIENCE
 AUTH0_MGMT_ORG_ID=RUNTIME_AUTH0_MGMT_ORG_ID
-VOTEX_MODULES=RUNTIME_VOTEX_MODULES
\ No newline at end of file
+VOTEX_MODULES=RUNTIME_VOTEX_MODULES
```
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/.env.production:6]: There is a typo here, pls rename `VOTEX_**` to `VORTEX_**`

File: vortex-app/vortex-app-portal/.env.production [hunk 2]
```diff
@@ -3,4 +3,15 @@ AUTH0_DOMAIN=RUNTIME_AUTH0_DOMAIN
...
+CUSTOMER_AUTH0_DOMAIN=RUNTIME_CUSTOMER_AUTH0_DOMAIN
+CUSTOMER_AUTH0_CLIENT_ID=RUNTIME_CUSTOMER_AUTH0_CLIENT_ID
+CUSTOMER_AUTH0_MGMT_ORG_ID=RUNTIME_CUSTOMER_AUTH0_MGMT_ORG_ID
+CUSTOMER_AUTH0_AUDIENCE=RUNTIME_CUSTOMER_AUTH0_AUDIENCE
+THEME_COLOR=RUNTIME_THEME_COLOR
+COMPANY_LOGO_URL=RUNTIME_COMPANY_LOGO_UR
```
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/.env.production:12]: `L` missed in `RUNTIME_COMPANY_LOGO_UR`

File: vortex-app/vortex-app-portal/.env.production [hunk 3]
```diff
@@ -3,4 +3,15 @@ AUTH0_DOMAIN=RUNTIME_AUTH0_DOMAIN
...
+SPECIFICATIONS_LINK_URL=RUNTIME_SPECIFICATIONS_LINK_URL
+HELP_CENTER_URL=RUNTIME_HELP_CENTER_URL
+TERM_OF_USE_URL=RUNTIME_TERM_OF_USE_URL
+PRIVACY_STATMENT_URL=RUNTIME_PRIVACY_STATMENT_URL
+COOKIE_URL=RUNTIME_COOKIE_UR
```
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/.env.production:17]: `L` missed in `RUNTIME_COMPANY_LOGO_UR`

File: vortex-app/vortex-app-portal/.env.production [hunk 4]
```diff
@@ -3,4 +3,15 @@ AUTH0_DOMAIN=RUNTIME_AUTH0_DOMAIN
...
+CUSTOMER_AUTH0_DOMAIN=RUNTIME_CUSTOMER_AUTH0_DOMAIN
+CUSTOMER_AUTH0_CLIENT_ID=RUNTIME_CUSTOMER_AUTH0_CLIENT_ID
+CUSTOMER_AUTH0_MGMT_ORG_ID=RUNTIME_CUSTOMER_AUTH0_MGMT_ORG_ID
+CUSTOMER_AUTH0_AUDIENCE=RUNTIME_CUSTOMER_AUTH0_AUDIENCE
+THEME_COLOR=RUNTIME_THEME_COLOR
+COMPANY_LOGO_URL=RUNTIME_COMPANY_LOGO_UR
+SPECIFICATIONS_LINK_URL=RUNTIME_SPECIFICATIONS_LINK_URL
+HELP_CENTER_URL=RUNTIME_HELP_CENTER_URL
+TERM_OF_USE_URL=RUNTIME_TERM_OF_USE_URL
+PRIVACY_STATMENT_URL=RUNTIME_PRIVACY_STATMENT_URL
```
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/.env.production:16]: `STATMENT` --> `STATEMENT`

Comment (yangpangpangya) [reply vortex-app/vortex-app-portal/.env.production]: have updated

**Review by btngoc-cmcglobal — APPROVED** [review]

**Review by btngoc-cmcglobal — COMMENTED** [review]
approved

**Review by btngoc-cmcglobal — APPROVED** [review]

**Review by btngoc-cmcglobal — APPROVED** [review]

**Review by DaveXiong — APPROVED** [review]
LGTM

**Pull request merged**